tests/test_main.py

```diff
@@ -1,125 +1,20 @@
-from unittest.mock import patch
-import pytest
-import os
+# tests/test_main.py
+import unittest
+from car_wash_station import CarWashStation  # Импортируем модуль из корня проекта
 
-from app.main import Car, CarWashStation
+class TestCarWashStation(unittest.TestCase):
+    def setUp(self):
+        self.station = CarWashStation()
```

Comment:
The `CarWashStation` class constructor is missing required parameters. Ensure to pass `station_id`, `location_id`, `avg_rating`, and `num_ratings` when creating an instance.